client/src/components/Places/index.js

```diff
@@ -10,19 +10,28 @@ class Places extends React.Component {
   }
 
   componentDidMount() {
-    this.setState({ initialList: this.props.listOfItems, filteredItems: this.props.listOfItems })
+    const itemList = sessionStorage.getItem('places')
+    const dataParsed = JSON.parse(itemList)
+    console.log('list', dataParsed)
+
+    this.setState({
+      initialList: dataParsed,
```

Comment:
name here should be consistent with Guides.
```suggestion
      initialList: parsedData,
```